contrib/cirrus/coverage.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+source $(dirname $0)/lib.sh
+
+req_env_var GOSRC
```

Comment:
I would recommend adding 'CODECOV_TOKEN' to the line above.

Reply:
Done, thanks!